nilearn/plotting/img_plotting.py

```diff
@@ -963,14 +963,16 @@ def plot_glass_brain(stat_map_img,
             See http://nilearn.github.io/manipulating_visualizing/manipulating_images.html#niimg.
             The statistical map image. It needs to be in MNI space
             in order to align with the brain schematics.
-        output_file : string, or None, optional
+        output_file : string, or None, optional.
             The name of an image file to export the plot to. Valid extensions
             are .png, .pdf, .svg. If output_file is not None, the plot
             is saved to a file, and the display is closed.
-        display_mode: {'ortho', 'x', 'y', 'z', 'xy', 'xz', 'yz'}
+        display_mode : {'ortho', 'x', 'y', 'z', 'xz', 'yx', 'yz', 'l', 'r',
+                        'lr', 'lzr', 'lyr', 'lzry', 'lyrz'}. Default is 'ortho'.
```

Comment:
Your problem with circleCI might be that you cannot have a line return here. It's possible that this just needs to be a very long line.